backend/pkg/config/config.go

```diff
@@ -86,6 +87,32 @@ func Parse(args []string) (*Config, error) {
 		return nil, fmt.Errorf("error parsing flags: %w", err)
 	}
 
+	// Handle list-plugins arg
```

Comment:
Instead of in the config module, this code should probably set the config.ListPlugins true here, and a ListPlugins function could be called inside `server.go@main`.

All the code related to plugins should generally go in the plugins folder. If it needs to be called from elsewhere then call the function from the plugins module. So that we can easily find where plugins related code is.

Reply:
Done